llvm/test/Transforms/SimplifyCFG/RISCV/switch-of-powers-of-two.ll

```diff
@@ -0,0 +1,453 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --tool ./bin/opt --version 3
+; RUN: opt -passes='simplifycfg<switch-to-lookup>' -simplifycfg-require-and-preserve-domtree=1 -S -mtriple=riscv64 < %s \
+; RUN: | FileCheck %s --check-prefixes=CHECK,RV64I
+; RUN: opt -passes='simplifycfg<switch-to-lookup>' -simplifycfg-require-and-preserve-domtree=1 -S -mtriple=riscv64 -mattr=+zbb < %s \
+; RUN: | FileCheck %s --check-prefixes=CHECK,RV64ZBB
```

Comment:
This causes distinct check lines for all tests due to the extra function attributes. Can we avoid that somehow? Maybe the easiest way is to run update_test_checks.py with `--version 1` to avoid inclusion of the function signature. Or maybe if you add an explicit `-mattr=-zbb` to the first one that would also avoid this difference?

Reply:
> This causes distinct check lines for all tests due to the extra function attributes. Can we avoid that somehow? Maybe the easiest way is to run update_test_checks.py with `--version 1` to avoid inclusion of the function signature. Or maybe if you add an explicit `-mattr=-zbb` to the first one that would also avoid this difference?

Adding `--version 1` helps. Thanks